chesster.js

```diff
@@ -3,6 +3,7 @@ var Botkit = require('botkit');
 var GoogleSpreadsheet = require("google-spreadsheet");
 var fs = require('fs');
 var fuzzy = require('./fuzzy_match.js');
+var spreadsheets = require('./spreadsheets.js');
 
 var SHEET_URL = "https://lichess4545.slack.com/files/mrlegilimens/F0VNACY64/lichess4545season3-graphs";
```

Comment:
these should be configurable as well. Ill work on that eventually as well. I need to find some time to do some programming.